src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -116,8 +112,7 @@ internal static class LoopFilter
         // 64 bit mask to shift and set for each prediction size. A bit is set for
         // each 8x8 block that would be in the left most block of the given block
         // size in the 64x64 block.
-        private static readonly ulong[] SizeMask = new ulong[]
-        {
+        private static readonly ulong[] _sizeMask = {
             0x0000000000000001UL,  // BLOCK_4X4
```

Comment:
```suggestion
            0x0000000000000001UL, // BLOCK_4X4
```
Same for other comments